### ISSUE TITLE
[BUG] There is no "value" key in the evaluation response

### ISSUE THREAD

**Issue by zeezdev** [open-feature/flagd#1116]
### Observed behavior

When a request to a new endpoint `flagd.evaluation.v1.Service/ResolveBoolean` (https://github.com/open-feature/flagd/pull/1083) for a boolean flag evaluates to a negative variant, then the response is missing the `value` key:

```json
{"reason":"TARGETING_MATCH", "variant":"off", "metadata":{}}
```

### Expected Behavior

The evaluation response contains `value` key for both negative and positive variants:
```json
{"value":false, "reason":"TARGETING_MATCH", "variant":"off", "metadata":{}}
```

```json
{"value":true, "reason":"TARGETING_MATCH", "variant":"on", "metadata":{}}
```

### Steps to reproduce

The flags definition (`test.json`):
```json
{
  "flags": {
    "EXPERIMENT__CREATE": {
      "variants": {
        "on": true,
        "off": false
      },
      "state": "ENABLED",
      "defaultVariant": "off",
      "targeting": {
        "if": [
          {
            "==": [
              false,
              {
                "var": [
                  "user.is_superuser"
                ]
              }
            ]
          },
          "on",
          "off"
        ]
      }
    }
  }
}
```

Up `flagd` (v0.8.0):
```
docker run \
  --rm -it \
  --name flagd \
  -p 8013:8013 \
  -v $(pwd):/etc/flagd/ \
  ghcr.io/open-feature/flagd:v0.8.0 \
  start --uri file:/etc/flagd/test.json --debug
```

Request for a new endpoint (`flagd.evaluation.v1.Service/ResolveBoolean`) with a context for which the variant `off` is expected:

```bash
curl -X POST 'http://localhost:8013/flagd.evaluation.v1.Service/ResolveBoolean' \
--header 'Content-Type: application/json' \
--data '
{
	"flagKey": "EXPERIMENT__CREATE",
	"context": {
		"user": {
			"is_superuser": true
		}
	}
}'
```

The response does not contain `value` attribute:
```bash
{"reason":"TARGETING_MATCH", "variant":"off", "metadata":{}}
```

Request for a new endpoint (`flagd.evaluation.v1.Service/ResolveBoolean`) with a context for which the variant `on` is expected:

```bash
curl -X POST 'http://localhost:8013/flagd.evaluation.v1.Service/ResolveBoolean' \
--header 'Content-Type: application/json' \
--data '
{
	"flagKey": "EXPERIMENT__CREATE",
	"context": {
		"user": {
			"is_superuser": false
		}
	}
}'
```

The `value` attribute is present in the response:
```bash
{"value":true, "reason":"TARGETING_MATCH", "variant":"on", "metadata":{}}
```

When a request is made to the old endpoint by both contexts (`off` & `on` variants are expected), the response contains the `value` attribute:

```bash
curl -X POST 'http://localhost:8013/schema.v1.Service/ResolveBoolean' \
--header 'Content-Type: application/json' \
--data '
{
	"flagKey": "EXPERIMENT__CREATE",
	"context": {
		"user": {
			"is_superuser": false
		}
	}
}'
```

```bash
{"value":true, "reason":"TARGETING_MATCH", "variant":"on", "metadata":{}}
``` 

```bash
curl -X POST 'http://localhost:8013/schema.v1.Service/ResolveBoolean' \
--header 'Content-Type: application/json' \
--data '
{
	"flagKey": "EXPERIMENT__CREATE",
	"context": {
		"user": {
			"is_superuser": true
		}
	}
}'
```

```bash
{"value":false, "reason":"TARGETING_MATCH", "variant":"off", "metadata":{}}
```

**Comment by beeme1mr** [comment]
Hey @zeezdev, thanks for reporting this. I'll work on a fix right await.

**Comment by beeme1mr** [comment]
Fixed in [v0.8.1](https://github.com/open-feature/flagd/releases/tag/flagd%2Fv0.8.1)
